### DISCUSSION TITLE
Flatpak apps not show in the Launcher

### DISCUSSION THREAD

**Discussion by vampywiz17** [LingmoOS/Community#11] [Bugs Report/缺陷提交] [answered]
If i install a flatpak app, for example Chrome, i able to start it on Terminal, but not see it in Launcher.

**Comment by elysia-best** [comment, marked as answer]
This feature is currently not supported by the launcher app. It's worth adding this feature to it. Thanks for the advise.

**Comment by elysia-best** [comment]
Moved to https://github.com/LingmoOS/lingmo-launcher/issues/3
